src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Dataplane/Commands/Synchronize-AzureASInstance.cs

```diff
@@ -42,7 +42,7 @@ public class SynchronizeAzureAzureAnalysisServer : AzurePSCmdlet
     {
         private static TimeSpan DefaultPollingInterval = TimeSpan.FromSeconds(30);
```

Comment:
We don't need to set this to 0 in the tests?

Reply:
Nope it isn't used in any tests.